api/README.md

```diff
@@ -0,0 +1,612 @@
+[back to hoodie-store-server](../README.md)
+
+# hoodie-store-server/api
+
+> Store API to manage databases and data.
+
+After registering the `@hoodie/store-server` hapi plugin, the `Store` API becomes available at `server.plugins.store.api`, so for example to create a new database with a plugin, use `server.plugins.store.api.create('mydb')` It can also be required directly.
+
+## Example
```

Comment:
it’s an example of requiring the API directly. We could make it in theory its own package, but I don’t think it’s worth it at this point